src/Total.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './App.css';
+
+class Total extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      total: 0
+    }
+  }
+
+  componentDidMount() {
+    let total = this.state;
+    this.setState({
+      total: this.props.price += this.props.shippingCost
+    })
+  }
+
+  render() {
+    return (
+      <div>
+        <p>{this.props.price}</p>
+        <p>+{this.props.shippingCost}</p>
+        <hr />
+        <p>={this.state.total}</p>
```

Comment:
Can we make this = sign a : instead to match the mocks? Otherwise, everything looks good!

Reply:
@NoahDavidATL you need the = in order to form the math on strings